README.md

```diff
@@ -588,18 +655,20 @@ The parameters passed to the function are:
 
 
 
-##### `set`
+#### `set`
 
 A function that is called whenever the property is set. This is also called when the property is first initialised.
 
 ```js
-skate.define('my-component', {
-  props: {
-    myProp: {
-      set (elem, data) {
-        // do something
+customElements.define('my-component', class extends skate.Component {
+  static get props () {
+    return {
+      myProp: {
+        set (elem, data) {
+          // do something
```

Comment:
I'm not sure what to do here. The line you've commented on says `// do something` and that PR comment is about clearing up if prop functions are a black box or syntactic sugar.